src/Symfony/Component/Console/Tests/Command/LockableTraitTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Tests\Command;
+
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Filesystem\LockHandler;
+
+class LockableTraitTest extends \PHPUnit_Framework_TestCase
+{
+    protected static $fixturesPath;
+
+    public static function setUpBeforeClass()
+    {
+        self::$fixturesPath = __DIR__.'/../Fixtures/';
+        require_once self::$fixturesPath.'/FooLockCommand.php';
+    }
+
+    public function testLockIsReleased()
+    {
+        if (!class_exists('Symfony\Component\Filesystem\LockHandler')) {
+            $this->markTestSkipped();
+        }
```

Comment:
These conditions must be removed and a dev requirement on filesystem added in composer.json instead.